### ISSUE TITLE
Get transparency instead of white opaque background / inference_webcam

### ISSUE THREAD

**Issue by arthurmchr** [PeterL1n/BackgroundMattingV2#55]
I am trying to replace the white color of the `inference_webcam.py` background example by transparency.

So far, I have replaced `pha * fgr + (1 - pha) * torch.ones_like(fgr)` by `torch.cat([pha.ne(0) * fgr, pha], dim=1)` and `cv2.imencode('.jpg', res)` by `cv2.imencode('.png', res)` but it still not works.

Any suggestions ?

Thanks a lot.

**Comment by PeterL1n** [comment]
I didn't test the below code so it may have bugs but I think it might work.

```
res = torch.cat([pha.ne(0) * fgr, pha], dim=1)
res = res[0]
res = res.detach().cpu().permute(1, 2, 0).mul_(255).byte().numpy()
res = cv2.cvtColor(res, cv2.COLOR_RGB2BGR)
res = cv2.imencode('.png', res)
```

**Comment by arthurmchr** [comment]
Thanks for your suggestion. Saddly it does not work, I got a black background and pixels do not seem to got alpha (only full opacity or none, not any intermediate)
